Tests/SwiftDocCTests/Rendering/PageKindTests.swift

```diff
@@ -118,4 +118,44 @@ class PageKindTests: XCTestCase {
             XCTAssert(problems.isEmpty)
         }
     }
+    
+    // Verify that we assign the `Collection` role to the root article of a
+    // documentation catalog that contains only one article.
+    func testRoleForSingleArticleCatalog() throws {
```

Comment:
These two tests are identical, except for the name of the bundle. Can you extract a local, private function, call that from each test maybe passing the bundle name as an argument, and avoid all the duplication?

Also, most the tests above in this file are also very similar. Maybe we should take a bit of extra time and create a shared setup function for use across PageKindTests.swift to:
- load a test bundle,
- look up an article and
- create a render node for it.

It could take the bundle name and path as arguments. Then each test just has one or two assertions. What do you think?

Reply:
I did some refactoring in this file. Created the function `generateRenderNodeFromBundle` that encapsulates most of the shared behaviour of these tests.
 
Resolved in https://github.com/apple/swift-docc/pull/744/commits/e6d326e31d0f952a2175705a25d36d9d1d8ce1ec.

Thanks @patshaughnessy !